### PR TITLE
Adds labels and milestones display to PR

### PR DESCRIPTION
**Pull request by VincentCATILLON**
Adds display of labels and milestones (separately) from Github Pull Requests:
- Displays the label inside a badge with his name as content, and his color as background-color
- Displays the milestone inside a badge with a background-color at white, and color at black (to the right)
- You can disable them in config.json inside your team config (default: enabled)

### CODE REVIEW

**Comment by fdubost** [comment]
:v: very useful ! However, tests have to be fixed :wink:

**Comment by VincentCATILLON** [comment]
Hi,

Thx for your code review.
The failed status is present since you merged the PR #25 3 months ago, as we can see it in the Travis build history tab : https://travis-ci.org/M6Web/GithubTeamReviewer/builds (build #113).

IMHO it depends on the Travis job config (comparing it to the build #112 that succeed and flagged as deprecated).

**Comment by VincentCATILLON** [comment]
I added the milestone support (pull at the right, before the repo name).
I don't know how to fix the Travis job errors, because i think it's related to the npm depedencies.

If you have answers about build errors, please come back to me to fix them :+1:

**Comment by VincentCATILLON** [comment]
Ping @fdubost .
Can you reply to my comment above, above the "failed" status since you merged the PR #25 ?

**Comment by fdubost** [comment]
Hi @VincentCATILLON,
Sorry for the delay 😊 
I have just fixed tests in #27, could you rebase your branch on master ?

**Comment by VincentCATILLON** [comment]
Hi @fdubost ,

It's ok about the delay, i'm not in a hurry :+1: 
Of course, it's done :)

File: README.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ Options :
   * *org* : an array of Github organizations,
   * *apiUrl* : url of your Github API (optional, default is `https://api.github.com`),
   * *descendingOrder* : allow to change ordering of pull requests (optional, default is `true`).
+  * *labels* : display labels of pull requests (optional, default is `true`).
```
Comment (fdubost) [inline README.md]: I would prefer `false` by default 👼

File: README.md [hunk 2]
```diff
@@ -36,6 +36,8 @@ Options :
...
+  * *milestones* : display milestones of pull requests (optional, default is `true`).
```
Comment (fdubost) [inline README.md]: Here too.

File: app/scripts/services/pullFetcher.js [hunk 3]
```diff
@@ -71,10 +71,26 @@ angular.module('gtrApp')
...
+      };
+
       var getRepoPulls = function (repo) {
         return request(repo.pulls_url.replace('{/number}', ''))
           .then(function (response) {
             var filtered = response.data.filter(filterPulls);
+            if (currentTeam.labels) {
+              filtered.map(addLabelsToPull);
+            }
+            if (!currentTeam.milestones) {
```
Comment (fdubost) [inline app/scripts/services/pullFetcher.js]: `if (currentTeam.milestones) {` without `!`, no ? 👀

**Comment by fdubost** [comment]
Could you add some tests for this feature ? 🐤

File: app/scripts/services/pullFetcher.js (hunk 3 above)
Comment (VincentCATILLON) [inline app/scripts/services/pullFetcher.js]: Yes, the milestones are already in the API response. I just added the removal when it's disabled.

**Comment by VincentCATILLON** [comment]
Hi @fdubost,
The default options have been updated according to your review and i updated the unit tests.

File: app/scripts/services/pullFetcher.js (hunk 3 above)
Comment (fdubost) [inline app/scripts/services/pullFetcher.js]: OK, I didn't see that it is a "remove" 😄

**Comment by fdubost** [comment]
👍  thank you for you contribution 😉

**Comment by VincentCATILLON** [comment]
You're welcome 👍

**Pull request merged**